tests/test_jinja.py

```diff
@@ -61,6 +61,233 @@ def test_jinja_platform(monkeypatch):
     assert render_jinja(template, filename="", is_included=False) == platform.python_revision()
 
 
+def test_jinja_x86(monkeypatch):
+    template = "{{ x86 }}"
+
+    platform_bkp = platform
```

Comment:
I need help here to mock this properly. I need to enforce `platform.machine()` returns a given value (e.g., `'x86'`)

Reply:
You can use `monkeypatch.setattr(platform, "machine", lambda: 'x86')` and let pytest revert it back for yoy; it is fine to call `monkeypatch.setattr` multiple times too.

Reply:
Hmmm I was almost there, but doing `monkeypatch.setattr(platform, "machine", 'x86')` instead, which caused problems later.